tests/rules/prefix-selectors-with-select.test.ts

```diff
@@ -24,48 +25,175 @@ ruleTester().run(path.parse(__filename).name, rule, {
     export const selectFeature = createSelectorFactory(factoryFn)`,
   ],
   invalid: [
-    fromFixture(
-      stripIndent`
-        export const getCount: MemoizedSelector<any, any> = (state: AppState) => state.feature
-                     ~~~~~~~~                        [${messageId}]`,
-    ),
-    fromFixture(
-      stripIndent`
-        export const getFeature = createSelector((state: AppState) => state.feature)
-                     ~~~~~~~~~~                        [${messageId}]`,
-    ),
-    fromFixture(
-      stripIndent`
-        export const select = (id: string) => createSelector(selectThings, things => things[id])
-                     ~~~~~~                            [${messageId}]`,
-    ),
-    fromFixture(
-      stripIndent`
-        export const testSelect = (id: string) => {
-                     ~~~~~~~~~~                        [${messageId}]
-          return createSelector(selectThings, things => things[id])
-        }`,
-    ),
-    fromFixture(
-      stripIndent`
-      export const feature = createFeatureSelector<AppState, FeatureState>(featureKey)
-                   ~~~~~~~                             [${messageId}]`,
-    ),
-    fromFixture(
-      stripIndent`
-        export const selectfeature = createSelector((state: AppState) => state.feature)
-                     ~~~~~~~~~~~~~                     [${messageId}]`,
-    ),
-    fromFixture(
-      stripIndent`
-        export const createSelect = createSelectorFactory((projectionFun) =>
-                     ~~~~~~~~~~~~                     [${messageId}]
-          defaultMemoize(
-            projectionFun,
-            orderDoesNotMatterComparer,
-            orderDoesNotMatterComparer,
-          ),
-        )`,
-    ),
+    {
+      code: stripIndent`
```

Comment:
The bad with suggestions is that the tests become really verbose with this big `errors` array 😞